src/components/[guild]/Requirements/components/GuildCheckout/components/buttons/ChooseCurrencyButton.tsx

```diff
@@ -0,0 +1,12 @@
+import Button from "components/common/Button"
+import CardMotionWrapper from "components/common/CardMotionWrapper"
+
+const ChooseCurrencyButton = (): JSX.Element => (
```

Comment:
This was confusing for me at first, would like it better if it'd be inlined in GuildCheckout, as the only button that's just a disabled placeholder